docs/integrations/dbt.md

```diff
@@ -22,6 +24,56 @@ Models **require** a start date for backfilling data through use of the `start`
 >   +start: Jan 1 2000
 ```
 
+
+### Runtime vars
+
+dbt supports passing variable values at runtime with its [CLI `vars` option](https://docs.getdbt.com/docs/build/project-variables#defining-variables-on-the-command-line).
+
+In SQLMesh, these variables are passed via configurations. When you initialize a dbt project with `sqlmesh init`, a file `config.py` is created in your project directory. The file contents are:
+
+```python
+from pathlib import Path
+
+from sqlmesh.dbt.loader import sqlmesh_config
+
+config = sqlmesh_config(Path(__file__).parent)
+```
```

Comment:
Do we want to include the whole file or just the line that matters? Rationale being that the file created by init could change some over time.